public/common/types/layer.ts

```diff
@@ -42,6 +45,9 @@ export const DEFAULT_MAP_EXPLORER_VIS_PARAMS: MapsExplorerVisParams = {
       }),
       layerType: LayerTypes.TMSLayer,
       isDesatured: false,
+      isHidden: false,
+      minZoom: 0,
+      maxZoom: 14,
```

Comment:
let's replace all these numbers with the same constants across the files : )

Reply:
- Will create the suggested constants file